idemp-rest/WEB-INF/src/org/bandahealth/idempiere/rest/service/db/PaymentDBService.java

```diff
@@ -116,7 +116,13 @@ public Payment saveEntity(Payment entity) {
 		}
 
 		if (entity.getPaymentType() != null) {
```

Comment:
I'm guessing this change will also break the payment form since it submits the value. Can you update that so it still works?